src/components/hall-of-fame-table-component.js

```diff
@@ -0,0 +1,88 @@
+import React from "react"
+
+export default function HallOfFameTableComponent({
+  person: { name, handle, handleLink, highestRating, level },
+}) {
+  const colors = {
+    Pupil: "text-green-600",
+    Master: "text-amber-400",
+    Grandmaster: "text-red-500",
+    Expert: "text-blue-700",
+    Specialist: "text-[#66CDAA]",
+    CandidateMaster: "text-fuchsia-600",
+  }
+
+  const codechefColors = {
+    2: "#008000",
+    3: "blue",
+    4: "purple",
+    5: "yellow",
+    6: "#FFA500",
+    7: "#800000",
+  }
+
+  const codechefColors2 = {
+    2: "text-[#008000]",
+    3: "text-blue-500",
+    4: "text-purple-600",
+    5: "text-yellow-300",
+    6: "text-[#FFA500]",
+    7: "text-[#800000]",
+  }
+  return (
+    <tr>
+      {name ? (
+        <td className="px-1 py-0 sm:px-5 sm:py-1 border border-white border-slate-100 text-center text-white">
+          {name}
+        </td>
+      ) : null}
+
+      {colors[level] ? (
+        <td
+          className={`sm:px-5 sm:py-1 border border-white border-slate-100 text-center font-semibold ${colors[level]}`}
```

Comment:
Create a class in global css for `sm:px-5 sm:py-1 border border-white border-slate-100 text-center font-semibold `